### PR TITLE
Feat: Added Product Page

### PR DESCRIPTION
**Pull request by AshwiniSetty**
This is the Product page listing with different filters.

image

Preview: 
https://deploy-preview-3--minwii-stores.netlify.app/

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      display: flex;
+      justify-content: space-between;
+  }
+
+.doc-content{
+    min-width: 0;
+    margin: 0px 2rem;
+    max-width: 90%;
+    margin-left: 0%;
+    margin-right: 0%;
```
Comment (adith-a-danthi) [inline css/main.css:51]: You can simplify the margin code to `margin: 0;`

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 5px;
+}
+
+.card-vertical {
+    max-width: 300px;
+    width: 100%;
```
Comment (adith-a-danthi) [inline css/main.css:61]: This can be simplified to `width: 300px`

File: pages/product_page.html [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+            <button class="btn-icon"><i class="fa-solid fa-cart-shopping"></i></button>
+            <span class="badge-status round-badge badge-notification primary">4</span>
+        </div>
+
+        <button class="btn"><i class="fa-solid fa-user"></i>Account</button>
+        
+    </div>
+</nav>
+<div class="doc-container">
+    <div class="side-bar">
```
Comment (adith-a-danthi) [inline pages/product_page.html:40]: You can use `<aside>` tag for sidebar

File: pages/product_page.html [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+            <div class="card card-vertical">
+                <div class="card-main">
+                    <div class="card-header">
+                        <button class="card-btn">
+                            <span>
+                                <i class="far fa-heart"></i>
+                            </span>
+                        </button>
+                        
+                        <img src="/assets/images/product-page-image.jpg" alt=" " class="card-img img-responsive">
```
Comment (adith-a-danthi) [inline pages/product_page.html]: You can fill up `alt` attributes of the image tags.

**Review by adith-a-danthi — COMMENTED** [review]
Looks Good. Clean, well-formatted code.

**Review by JaswanthReddyPadala — CHANGES_REQUESTED** [review]
Add alt for images, 
And then you are good to go

File: pages/product_page.html (hunk 3 above)
Comment (AshwiniSetty) [reply pages/product_page.html:40]: Yes will keep in mind to use the aside tag

File: pages/product_page.html (hunk 4 above)
Comment (AshwiniSetty) [reply pages/product_page.html]: updated the alt for images

**Comment by AshwiniSetty** [comment]
Updated the alt for images

File: css/main.css (hunk 2 above)
Comment (AshwiniSetty) [reply css/main.css:61]: I actually did not get it, max width is 300 px and less than that will be 100%. That is what I wanted

**Review by JaswanthReddyPadala — APPROVED** [review]
Good to go ✅

**Review by khushijohri2001 — COMMENTED** [review]
You should give some space between wishlist, cart and account button. Otherwise the code looks fine to me

File: css/main.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    height: max-content;
+}
+
+.card-btn{
```
Comment (khushijohri2001) [inline css/main.css:65]: You can remove extra spaces in the code

Comment (AshwiniSetty) [reply css/main.css:65]: Yes, fixed it. Thank you for the review.

**Pull request merged**